src/components/ContactUs/index.tsx

```diff
@@ -6,77 +6,61 @@ import { faEnvelope, faPhoneAlt } from '@fortawesome/free-solid-svg-icons';
 import { faFacebook, faInstagram } from '@fortawesome/free-brands-svg-icons';
 
 const ContactUsContainer = styled.div`
-display: flex;
-flex-flow: column nowrap;
-padding: 3rem .5rem;
-align-items: center;
-justify-content: center;
-width: 75%;
-margin: 0 auto;
+display:flex;
+flex-flow:row wrap;
+width: 85%;
+margin: 2rem auto;
+padding-bottom:2rem;
 border-radius: 10px;
-margin-bottom: 5rem;
-background-color: #F5F5F5;
+background:#EBEBEB;
 font-weight:700;
+box-shadow:2px 5px 5px #888888;
+${mediaQueries('tablet')`
+  position:relative;
+  top:-2rem;
+  margin: 0 auto`}
+${mediaQueries('laptop')`
+flex-flow: row nowrap;
```

Comment:
why the 'nowrap'? what would happen if you add a new service?

Reply:
not being a designer, I would doubt you would need any other new contact option tbh...?